src/Protocol/Diaspora.php

```diff
@@ -1441,17 +1445,17 @@ private static function storeMentions(int $uriid, string $text)
 	/**
 	 * Processes an incoming comment
 	 *
-	 * @param array  $importer Array of the importer user
-	 * @param string $sender   The sender of the message
-	 * @param object $data     The message object
-	 * @param string $xml      The original XML of the message
-	 * @param bool   $fetched  The message had been fetched and not pushed
+	 * @param array  $importer  Array of the importer user
+	 * @param string $sender    The sender of the message
+	 * @param object $data      The message object
+	 * @param string $xml       The original XML of the message
+	 * @param int    $direction Indicates if the message had been fetched or pushed
```

Comment:
Please add a reference to the constants the parameter is taking its values from.